miscelanea/005-deploy-in-one-pod/freeipa.yaml.envsubst

```diff
@@ -0,0 +1,448 @@
+# Proof of Concept that launch ipa-server-install in an initContainer,
+# preparing the whole data volume.
+#
+# Quick Start:
+#   make app-delete app-deploy get-info
+#
+# References:
+#   https://systemd.io/CONTAINER_INTERFACE/
+#   https://stackoverflow.com/questions/52940774/kubernetes-how-to-check-current-domain-set-by-cluster-domain-from-pod
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: freeipa
+  labels:
+    app: freeipa
+data:
+  DEBUG_TRACE: "2"
+  IPA_SERVER_HOSTNAME: "${APP}.apps.${CLUSTER_DOMAIN}"
+  IPA_SERVER_IP: "10.0.135.189"
```

Comment:
We can remove this, and the commented reference to it below.

Reply:
Internal ticket: https://issues.redhat.com/browse/IDMOCP-186 . Resolving comment.